scripts/exportAllYamlCourses.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
 
+ANY_FAILED="0"
 for d in ./courses/*/ ; do
 		d=$(echo $d | sed 's/[^/]*\/[^/]*\///' | sed 's/\///')
 		./scripts/exportYamlCourse.sh $d
+		if [ "$?" -eq "1" ]; then
+			ANY_FAILED="1"
+		fi
 done
 
+if [ "$ANY_FAILED" -eq "1" ]; then
```

Comment:
This is a really cool new feature but right now it will always fail, right? So it might be a good idea to somehow separate this into a separate MR, and merge it once we've dealt with that issue (for example somehow temporarily disabled the courses that are breaking)

Reply:
Sounds good.